python/python/opm/io/ecl/__init__.py

```diff
@@ -0,0 +1,71 @@
+
+from opm._common import EclFileBind, eclArrType
+
+
+class EclFile:	
+
+    def __init__(self, fileName):
+        self.name = fileName
+        self.arrayNameList=[]
+        self.arrayTypeList=[]
+        
+        self.eclfile = EclFileBind(fileName)
+
+        arrayList = self.eclfile.getList()
+
+        for name, type, size in arrayList:
+            self.arrayNameList.append(name)
+
+            if type==eclArrType.INTE:
+                self.arrayTypeList.append(eclArrType.INTE)    
+            elif type==eclArrType.REAL:
+                self.arrayTypeList.append(eclArrType.REAL)    
+            elif type==eclArrType.DOUB:
+                self.arrayTypeList.append(eclArrType.DOUB)    
+            elif type==eclArrType.CHAR:
+                self.arrayTypeList.append(eclArrType.CHAR)    
+            elif type==eclArrType.LOGI:
+                self.arrayTypeList.append(eclArrType.LOGI)    
+            elif type==eclArrType.MESS:
+                self.arrayTypeList.append(eclArrType.MESS)
+            else:
+                message = "Unknown array type: %s for array: '%s' in file %s" % (repr(type),name, fileName) 
+                raise NameError(message)
+	
+    def getListOfArrays(self):
+        return self.arrayNameList	
+
+    def getNumArrays(self):
+        return len(self.arrayNameList)	
+
+    def hasArray(self, name):
+        return self.eclfile.hasKey(name)
+    
+    def get(self, arg):
```

Comment:
This should be moved to the C++ layer - but there it should be bound to `__getitem__`